sgt-back-end/index.js

```diff
@@ -0,0 +1,139 @@
+const express = require('express');
+const app = express();
+const pg = require('pg');
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+  select *
+    from "grades"
+  `;
+  db.query(sql)
+    .then(result => {
+      const grade = result.rows;
+      res.status(200).json(grade);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'an unexpected error occurred'
+      });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  const body = req.body;
+  if (!body.name || !body.course || !body.score) {
+    res.status(400).json({
+      error: "must have 'name', 'course', and 'score'."
+    });
+  }
+  if (Number(body.score) > 100 || Number(body.score) < 0) {
+    res.status(400).json({
+      error: 'score must be between 0 and 100'
+    });
+  }
```

Comment:
Be sure to use `else` or `return` where appropriate to prevent your server from attempting to respond to the same request more than once.  Responding does not halt the `app.METHOD()` callback's code block.